.changeset/early-bees-think.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes-backend': minor
```

Comment:
Not a breaking change right? Would make it `patch` in that case since we're on 0.x